modin/core/dataframe/algebra/default2pandas/groupby.py

```diff
@@ -116,14 +116,14 @@ def get_func(cls, grp, key, **kwargs):
 
         Notes
         -----
-        There is two ways of how groupby aggregation can be invoked:
+        There are two ways of how groupby aggregation can be invoked:
             1. Explicitly with query compiler method: `qc.groupby_sum()`.
             2. By passing aggregation function as an argument: `qc.groupby_agg("sum")`.
         Both are going to produce the same result, however in the first case actual aggregation
         function can be extracted from the method name, while for the second only from the method arguments.
         """
         if "agg_func" in kwargs:
-            return kwargs["agg_func"]
+            return cls.inplace_applyier_builder(key, kwargs["agg_func"])
```

Comment:
Previously, `kwargs["agg_func"]` was holding a function to be applied to a <i>groupby object</i>, now it stores a func to be applied to a <i>Series or a DataFrame</i> representing a certain group (specified via `qc.groupby_agg(how=...)` parameter), that's why we need to replace the direct "agg_func" return to the wrapper.